src/main/java/no/nav/pto/veilarbportefolje/pdldata/PdlDataService.java

```diff
@@ -0,0 +1,44 @@
+package no.nav.pto.veilarbportefolje.pdldata;
+
+import lombok.extern.slf4j.Slf4j;
+import no.nav.common.client.pdl.PdlClient;
+import no.nav.common.types.identer.AktorId;
+import no.nav.pto.veilarbportefolje.database.BrukerRepository;
+import no.nav.pto.veilarbportefolje.elastic.domene.OppfolgingsBruker;
+import no.nav.pto.veilarbportefolje.util.DateUtils;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+@Slf4j
+public class PdlDataService {
+    private final PdlRepository pdlRepository;
+    private final PdlClient pdlClient;
+    private final BrukerRepository brukerRepository;
+
+    @Autowired
+    public PdlDataService(PdlRepository pdlRepository, PdlClient pdlClient, BrukerRepository brukerRepository) {
+        this.brukerRepository = brukerRepository;
+        this.pdlRepository = pdlRepository;
+        this.pdlClient = pdlClient;
+    }
+
+    public void lastInnPdlData(AktorId aktorId){
+        String fodselsdag = pdlClient.rawRequest("fin fodselsdag på fnr");
```

Comment:
this rawRequest is bit unusual for me, how do you specify aktorId as input. Also what if fodselsdag cant be converted to Time..